docker-compose.override-example.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+services:
+  database:
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_USER: root
+      MYSQL_PASSWORD: root
+      MYSQL_DATABASE: testdb
+  web:
+    volumes:
```

Comment:
Should it though? Because as far as I understand it, docker-compose is kind of like a parent class that gets inherited by some subclasses in code and thus should only contain some overall basic knowledge that all the subvaritations want to be using! A production image for example wouldn't be using volumes so I think it shouldn't be forced to either!

With the current setup I believe I could quite easily just make a docker-`compose.production.yml` file which could be ran outside the dev environment and it wouldn't be using volumes because the `docker-compose.override.yml` wouldn't be declared!

This is the set up we're aiming at with `CircleCI` deploys, right?